reflection.md

```diff
@@ -2,8 +2,33 @@
 
 ## Which operators were easiest to implement and why?
 
+All of the operators were easier to implement than expected.  Scala seems
+to provide the coder with lots of freedom for choosing operators to define or
+overload.  I would say that the operators ||, ~, and <*> were the easiest to
+implement, since they were direct translations from the classes Union, Concat, and
+Star, respectively.  After this, <+> only involved Concat-ing `this` with 
+`Star(this)`
+
 ## Which operators were most difficult to implement and why?
 
+The most difficult to implement was probably the repetition operator.  The most
+difficult part was interpreting `'3'{7}` as a function of a literal that took an
+integer as an argument literal, and was equivalent to `myExpression(7)`.  After
+that, I had to do some googling before I found out that we were supposed to define
+a function `apply` that would work as parentheses.  Finally, implementing it
```

Comment:
I agree. This one took me a long time to do with extensive googling and weird stack overflow threads. In the end, I looked at the documentation for list, but I'm curious if there were other places that people found this answer for the problem.